CIPs/CIP-85/CIP-85.md

```diff
@@ -101,37 +105,33 @@ The `CollectionSlice` schema must be an `object` with a `$ceramic` field from [C
 
 #### Deterministic slice access
 
-Accessing slices in the collection relies on Ceramic's ability to load documents deterministically based on their genesis contents:
-
-- `collection`: the collection's `DocID` string
-- `sliceIndex`: the slice's index, an integer between `0` (first slice) and `slicesCount - 1` (last slice)
-- `contents`: empty array
+Accessing slices in the collection relies on Ceramic's ability to load streams deterministically based on their metadata by using `{ deterministic: true, tags: ['<slice tag>'] }`, where `slice tag` is a string of `<collection StreamID string>:<slice index>`.
```

Comment:
Are there any other tags we might want to include for the purpose of indexing services?  We can't change the tags we use later without breaking deterministic access.

For instance, I could imagine a tag just for the streamId of the parent collection, so indexers could find all slices for a given collection.

Reply:
I guess if we want to add tags later without breaking deterministic access we could do so they would just have to be added in the first update, not in the genesis commit

Reply:
Yes we need a first update in any case to add the contents so it shouldn't be a problem to also edit the metadata. I agree having a tag with the collection ID could be useful.

Reply:
will resolve this comment thread as it doesn't seem like this is actually a blocker since it can be changed later